python-template/{{cookiecutter.placeholder_repo_name}}/.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,39 +1,8 @@
-**Description:** Describe in a couple of sentences what this PR adds
```

Comment:
As an alternative to this PR: Why don't we delete this template from the cookiecutter, and instead add the PR template as an org-wide default to the `.github` repository? That way, updates to it will immediately apply to all repos without an explicit PR template, rather than only new repos.

Reply:
...I had no idea you could have an org-wide PR template! I actually really like that, because 1) I am generally suspicious of the maintenance burden that results from each new file the cookiecutter produces, and 2) it still allows customization if a repo has special requirements. I may open a new PR for `.github`.

Reply:
Nice. The only wrinkle is that not every repo has a changelog or versioning. But I'm sure you could figure out some creative wording to gloss over that.

Reply:
Speaking of that, how do you want folks to deal with check boxes that just don't apply to their PR?

Remove them? Strikethrough? Check them anyway?

Reply:
Maybe the org-wide one should end with "(If any of these are _never_ applicable to this repo, please make a copy of this org-wide pull request template and trim it as appropriate.)" :laughing: It's pretty hard to have something both comprehensive _and_ relevant.

For inapplicable ones I personally tend to just check them off in the spirit of keeping things low-effort. I've suggested the language "Check off if complete or not applicable:" which hopefully would disambiguate that in practice.

Reply:
> (If any of these are never applicable to this repo, please make a copy of this org-wide pull request template and trim it as appropriate.)

I like the sentiment but I fear that this would just take up space in the template and be ignored.

>  "Check off if complete or not applicable:" which hopefully would disambiguate that in practice.

How about "Delete if not applicable?" 

I think "I did this" versus "I don't think it's applicable" is a useful thing to communicate to reviewers. Particularly around documentation and manual testing.

Unless, of course, the purpose if this template is not to tell reviewers things, but is to remind authors to do things, in which case just checking the non-applicable boxes works fine.

Reply:
Yeah, more text is probably not the answer. :-P

I mostly think about it as reminding authors, although it's great for reviewers too. So I'd prefer ticking to deleting.